### ISSUE TITLE
Error with composer

### ISSUE THREAD

**Issue by czirkoszoltan** [yourivw/LEClient#30]
Bug #29 seems to still exist: if I do a `composer require yourivw/leclient 1.1.1`, I get
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for yourivw/leclient 1.1.1 -> satisfiable by yourivw/leclient[1.1.1].
    - yourivw/leclient 1.1.1 requires php ^5.2 -> your PHP version (7.0.28) does not satisfy that requirement.
```

There is another problem with the composer installation: when I do `composer require yourivw/leclient` (without specifying version), 1.1.0 gets installed. However, that one has
```
$headers = array('Accept: application/json', 'Content-Type: application/json');
```
at LEConnector.php(105), and does not work anymore.

**Comment by yourivw** [comment]
That is correct, since I haven't released it. #29 is corrected in the master branch, but not yet in the release. 
I'm aiming to make a new release shortly, which will fix the composer issue. I'll leave this issue open for clarity until then.

**Comment by HMAZonderland** [comment]
When are you planning on releasing it? It's been close to 2 months now?

**Comment by yourivw** [comment]
Made a release only resolving the composer error due to a lack of time to finish the further developments. Available now in [version 1.1.2](https://github.com/yourivw/LEClient/releases/tag/1.1.2).

Can anyone please confirm whether the installation works correctly now?

**Comment by czirkoszoltan** [comment]
Works for me!
